src/modules/wallet/ticket/Ticket.vue

```diff
@@ -66,7 +68,7 @@
         <base-icon-and-detail class="mb-1" :detail="time"
           ><ClockIcon
         /></base-icon-and-detail>
-        <base-icon-and-detail :detail="event.location.name"
+        <base-icon-and-detail v-if="event" :detail="event.location.name"

```

Comment:
Why v-if here?

Reply:
I forgot this one. Will remove it.

Reply:
I forgot krub, will remove v-if fron this line.